pkg/kev/project.go

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2021 Appvia Ltd <info@appvia.io>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kev
+
+import (
+	"github.com/appvia/kev/pkg/kev/terminal"
+)
+
+// Init initialises the base project to be used in a runner
+func (p *Project) Init(opts ...Options) {
+	var cfg runConfig
+	for _, o := range opts {
+		o(p, &cfg)
+	}
+	p.config = cfg
+	if p.UI == nil {
+		p.UI = terminal.ConsoleUI()
+	}
+}
+
+// Manifest returns the project's manifest
+func (p *Project) Manifest() *Manifest {
+	return p.manifest
+}
+
+func WithComposeSources(c []string) Options {
```

Comment:
@marcinc ... To clarify: 

Do you mean instead of passing `opts` to `Init()`. And, creating extra methods on `Project` to perform the wiring?

But how would you know which method (`.WithComposeSources(...)` or `.WithEvns`) to call?

Reply:
yeh, that's what I meant. Current implementation is correct, just thought that method chain could be slightly nicer. To your question, I think you'd know which method to chain the same way you know it now.. an excertp:

```go
	return kev.InitProjectWithOptions(wd,
		kev.WithComposeSources(files),
		kev.WithEnvs(envs),
		kev.WithSkaffold(skaffold))

# this would become

	return kev.InitProject(wd).WithComposeSources(files).WithEnvs(envs).WithSkaffold(skaffold)
```

Reply:
Tbh, I would rather leave it as is. 

Otherwise, you would need to:
-> `kev.InitProject(wd)` to return a runner object.
-> Configure using `With...` methods.

... And you also then:
-> have to `Run()` the runner.
-> and do the other things that `InitProjectWithOptions` is doing for you.

Reply:
as said, it's good as is. If chaining brings extra complexity then that's fine.